js/src/forum/components/NotificationGrid.js

```diff
@@ -19,21 +19,21 @@ export default class NotificationGrid extends Component {
     /**
      * Information about the available notification methods.
      *
-     * @type {Array}
+     * @type {({ name: string, icon: string, label: import('mithril').Children })[]}
      */
     this.methods = this.notificationMethods().toArray();
 
     /**
      * A map of which notification checkboxes are loading.
      *
-     * @type {Object}
+     * @type {Record<string, boolean>}
      */
     this.loading = {};
 
     /**
      * Information about the available notification types.
      *
-     * @type {Array}
+     * @type {({ name: string, icon: string, label: import('@askvortsov/rich-icu-message-formatter').NestedStringArray })[]}
```

Comment:
Same here, also should be Mithril.Children

Reply:
```suggestion
     * @type {({ name: string, icon: string, label: import('mithril').Children })[]}
```